recipe/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+redhat-lsb-core
```

Comment:
Had issues using the `gcc` package without this. Please see this PR ( https://github.com/conda-forge/docker-images/pull/34 ) for a more detailed write-up and potentially better proposal.